packages/core/src/common/utils/reactUtils.ts

```diff
@@ -26,8 +26,8 @@ import { isEmptyString } from "./jsUtils";
 export function isReactNodeEmpty(node?: React.ReactNode, skipArray = false): boolean {
     return (
         node == null ||
-        node === "" ||
         node === false ||
+        isEmptyString(node) ||
```

Comment:
I don't think we should consider " " an empty node - or does that not sound right? This seems like it has a chance to break something existing and I'd rather not try to verify all of the uses.

Reply:
fair fair. reverted.